client/history/client_test.go

```diff
@@ -33,13 +33,14 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"go.temporal.io/api/serviceerror"
+	"google.golang.org/grpc"
+
```

Comment:
actually the team decided to change our policy on imports and unify all non-stdlib imports in one big group. so you can remove this part